examples/lfric/eg17/README.md

```diff
@@ -0,0 +1,54 @@
+# PSyData Examples
+
+**Author:** J. Henrichs, Bureau of Meteorology
+
+This directory contains some runnable examples of LFRic. They are based
+on a subset of the LFRic infrastructure library, that has been slightly
+modified to make it easier to create stand-alone, non-MPI LFRic codes.
+The examples here show:
+- A [full_example](./full_example) of a stand-alone LFRic-based code. It
+  uses PSyclone for a very simple kernel.
```

Comment:
"It uses PSyclone for a very simple kernel" is a bit unclear. What does PSyclone do to the kernel? Does it generate the kernel, does it modify it or does it do something else (e.g. calls the kernel code on one column of data through the generated PSy layer)?

Also,
- What is the name of the kernel?
- How can the binary be created and run?
- What is the expected output of the binary run? 
- How can the workspace be cleaned?

**Edit:** I see that "Detailed instructions are in the various subdirectories." I assume you mean the relevant subdirectories in `eg17`? I checked the relevant `README`  and it could use some more information. I suggest adding the information requested for this bullet point (if it is not already there) to the `full_example/README.md`.

Reply:
> Also, running `make` on all `examples` fails due to the empty left-over directory `examples/lfric/full_example_extract` (needs to be removed).

I don't think I can fix that - git does not manage directories, it implicitly adds a directory when a file is added. In your case I'd guess you still have some compiled files in these directories (.o/.mod, ...), so git could not delete the directories when it updated; you have to do this manually.